google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITWriteTest.java

```diff
@@ -161,7 +183,7 @@ public static void setUpDatabase()
     if (!EmulatorSpannerHelper.isUsingEmulator()) {
       Database postgreSQLDatabase =
           env.getTestHelper()
-              .createTestDatabase(Dialect.POSTGRESQL, Collections.singletonList(POSTGRESQL_SCHEMA));
+              .createTestDatabase(Dialect.POSTGRESQL, Arrays.asList(POSTGRESQL_SCHEMA));
```

Comment:
Any specific reason for this?

Reply:
The schema was earlier a String. It's now a String[], after introduction of DDL for new table T1.

Reply:
Ah, of course. Thanks!